cmd/stencil/update_module.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+// Description: This file contains code for the create templaterepository command
+
+package main
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/getoutreach/stencil/pkg/configuration"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+	"gopkg.in/yaml.v3"
+)
+
+// NewUpdateModule returns a new urfave/cli.Command for the
+// update module command
+func NewUpdateModule() *cli.Command {
+	return &cli.Command{
+		Name:        "module",
+		Description: "updates a module with the provided name in the current directory",
+		ArgsUsage:   "update module",
+		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:  "native-extension",
+				Usage: "Updates a module to be a native extension",
+			},
+		},
+		Action: func(c *cli.Context) error {
+			readAndMergeServiceYaml("service.yaml", c.Bool("native-extension"))
+			//nolint:gosec // Why: intentional
+			cmd := exec.CommandContext(c.Context, os.Args[0])
```

Comment:
nit: I'd suggest adding a comment about what you're doing here. Took me a second to realize you're trying to call yourself